oe_contact_forms.module

```diff
@@ -6,3 +6,350 @@
  */
 
 declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\contact\ContactFormInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter() for contact_form_form().
+ */
+function oe_contact_forms_form_contact_form_form_alter(&$form, FormStateInterface $form_state): void {
+  /** @var \Drupal\contact\ContactFormEditForm $form_object */
+  $form_object = $form_state->getFormObject();
+
+  if (!in_array($operation = $form_object->getOperation(), ['edit', 'add'], TRUE)) {
+    // Only alter the edit and add forms.
+    return;
+  }
+
+  /** @var \Drupal\contact\ContactFormInterface $contact_form */
+  $contact_form = $form_object->getEntity();
+  $is_corporate_form = (boolean) ($form_state->getValue('is_corporate_form') ?? $contact_form->getThirdPartySetting('oe_contact_forms', 'is_corporate_form', FALSE));
+  $el = $form_state->getTriggeringElement();
```

Comment:
Let's not name please variables so short. Longer variable names preferred to short and unintelligible ones.

Reply:
`$el` replaced by `$element`

Reply:
$triggered_element